src/ui/components/molecules/network-login/network-login.scss

```diff
@@ -0,0 +1,120 @@
+@import '../../../../globals/variables';
+@import '../../../../globals/mixins';
+
+.panel {
+  background: $ads-background-main;
+  display: flex;
+  flex-direction: column;
+  height: 450px;
+  padding: 2.2rem;
+  width: 800px;
+
+  &--secondary {
+    background-color: $ads-secondary-main;
+    position: sticky;
+  }
+}
+
+.logo {
+  align-items: center;
+  background: transparent;
+  display: flexbox;
+  height: auto;
+  margin-inline: auto;
+  width: 400px;
+}
+
+.logo .logo-icon {
+  & svg {
+    height: 100%;
+    width: 100%;
+  }
+}
+
+.button-google {
+  margin-bottom: 10px;
+  margin-inline: auto;
+}
+
+.button-facebook {
+  margin-inline: auto;
+}
+
+.button__label {
+  border-radius: 20px;
+  height: auto;
+  width: 400px;
+}
+
+.button__text {
+  font-size: $ads-font-size-quarter;
+}
+
+.panel--secondary .button__label {
+  background-color: $ads-text-main-hover;
+}
+
+.panel--secondary .logo g {
+  fill: $ads-tertiary-main;
+}
+
+/* Pantalla grande (desktop) */
+@media (width >= 1200px) {
+  .panel {
+    width: 800px;
+  }
+
+  .logo {
+    width: 400px;
+  }
+
+  .button__label {
+    width: 400px;
+  }
+}
+
```

Comment:
no, it would help if you used the global mixes don't set your own media queries that is an antipattern